seed/static/seed/partials/mapping.html

```diff
@@ -148,15 +148,15 @@
                         </tr>
                     </thead>
                     <tbody id="mapped-table">
-                        <tr ng-repeat="col in mappings">
+                        <tr ng-repeat="col in mappings | filter:{suggestion: '!_source_filename'}">
```

Comment:
since `_source_filename` ends up a column mapping (see my other comments), we need to filter it out here so the user doesn't see it